src/main/java/internetshop/factory/Factory.java

```diff
@@ -18,52 +18,35 @@
 import internetshop.service.impl.UserServiceImpl;
 
 public class Factory {
-    private static ItemDao itemDao;
-    private static ItemService itemService;
-    private static BucketDao bucketDao;
-    private static BucketService bucketService;
-    private static OrderDao orderDao;
-    private static UserDao userDao;
-    private static OrderService orderService;
-    private static UserService userService;
-
-    public static BucketDao getBucketDao() {
-        return bucketDao != null ? bucketDao :
-                (bucketDao = new BucketDaoImpl());
+    public static ItemService getItemService() {
+        return new ItemServiceImpl();
     }
 
     public static BucketService getBucketService() {
-        return bucketService != null ? bucketService :
-                (bucketService = new BucketServiceImpl());
+        return new BucketServiceImpl();
```

Comment:
@p1q я б залишив попередній варіант
```suggestion
        return bucketService != null ? bucketService :
                (bucketService = new BucketServiceImpl());
```
І так у всіх методах цього класу